docs/content/installation/advanced_configuration/fds_mode.md

```diff
@@ -48,37 +183,24 @@ spec:
 
 {{< /highlight >}}
 
-{{% notice note %}}
-
-Note, Function Discovery needs to be enabled for this to work. See the next sections.
-
-{{% /notice %}}
-
-## Function Discovery Service (FDS)
+### How FDS works
 
 Using FDS means that the Gloo Gateway `discovery` component will make HTTP requests to all `Upstreams` known to Gloo Gateway trying to discover functions. This behavior causes increased network traffic and may be undesirable if it causes unexpected behavior or logs to appear in the services Gloo Gateway is attempting to poll. For this reason, we may want to restrict the manner in which FDS polls services.
 
 Gloo Gateway allows whitelisting/blacklisting services, either by namespace or on the individual service level.
 
 We can use these configuration settings to restrict FDS to discover only the namespaces or individual services we choose.
 
----
-
-## Configuring the `fdsMode` Setting
-
-FDS can run in one of 3 modes:
-
-* `BLACKLIST`: The most liberal FDS polling policy. Using this mode, FDS will poll any service unless its namespace or the service itself is explicitly blacklisted.
-* `WHITELIST`: A more restricted FDS polling policy. Using this mode, FDS will poll only those services who either live in an explicitly whitelisted namespace, or themselves are are explicitly whitelisted. *`WHITELIST` is the default mode for FDS*.
-* `DISABLED`: FDS will not run. **Upstream Discovery Service** (UDS) will still run as normal.
+### Enable FDS mode
 
-Setting the `fdsMode` can be done either via the Helm Chart, or by directly modifying the `default` `gloo.solo.io/v1.Settings` custom resource in Gloo Gateway's installation namespace (`gloo-system`).
+You can enable FDS mode with Helm or the Settings CR in the `gloo-system` namespace.
```

Comment:
same thing, maybe mention it is not enabled by default and needs to explicitly be turned on

Reply:
I have this note at the top of the page. Or do you think I should also reiterate it in both the H2s for UDS/FDS?

```
{{% notice note %}}
Because discovery is a resource-intensive process, both discovery services are disabled by default.
{{% /notice %}}
```